### PR TITLE
DOCS: Update version numbers and prereqs for 4x

### PR DESCRIPTION
**Pull request by ljamen**
### Description

This PR includes updates to the prerequisites and version numbers for 4.x variables in the doc. I used the pom.xml file to get the latest version numbers and then updated the attributes.adoc variables which are used throughout the the docs. 

### Documentation

If enhancement: provide description with example code/config snippet or pointer to issue with the description

If feature: summarize feature and provide pointer to doc issue

If no doc impact: None

### CODE REVIEW

**Comment by ljamen** [comment]
Note that this is how LRA is listed in the dependencies/pom.xml file. Does it need to be updated there? 
<version.lib.microprofile-lra-api>2.0-RC1</version.lib.microprofile-lra-api>

File: docs/includes/attributes.adoc [hunk 1]
```diff
@@ -35,53 +35,54 @@ ifeval::["{helidon-version-is-release}" == "true"]
 endif::[]
 
 // versions
-:version-plugin-helidon: 3.0.0-M3
+:version-plugin-helidon: 4.0.0-SNAPSHOT
```
Comment (barchetta) [inline docs/includes/attributes.adoc:38]: This should be "3.0.5" (it's a Helidon build tools version)

File: docs/includes/attributes.adoc [hunk 2]
```diff
@@ -35,53 +35,54 @@ ifeval::["{helidon-version-is-release}" == "true"]
...
 :version-lib-microprofile-graphql: 2.0
 :version-lib-microprofile-health: 4.0
-:version-lib-microprofile-jwt: 2.0
-:version-lib-microprofile-metrics-api: 4.0
-:version-lib-microprofile-openapi-api: 3.0
-:version-lib-microprofile-reactive-messaging-api: 3.0-RC2
-:version-lib-microprofile-rs-operators-api: 3.0-RC1
+:version-lib-microprofile-jwt: 2.1
+:version-lib-microprofile-metrics-api: 5.0.1
+:version-lib-microprofile-openapi-api: 3.1
```
Comment (barchetta) [inline docs/includes/attributes.adoc:50]: Now 3.1.1

**Review by barchetta — COMMENTED** [review]
A few dependencies recently changed.

File: docs/includes/attributes.adoc (hunk 2 above)
Comment (ljamen) [reply docs/includes/attributes.adoc:50]: line 52

**Review by barchetta — APPROVED** [review]

**Pull request merged**